streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java

```diff
@@ -370,41 +439,42 @@ private boolean testStoreMetricWindow(final StringBuilder errorMessage) {
             final List<Metric> listMetricStore = new ArrayList<Metric>(kafkaStreams.metrics().values()).stream()
                     .filter(m -> m.metricName().group().equals(STREAM_STORE_WINDOW_ROCKSDB_STATE_METRICS))
                     .collect(Collectors.toList());
-            testMetricByName(listMetricStore, PUT_LATENCY_AVG, 2);
-            testMetricByName(listMetricStore, PUT_LATENCY_MAX, 2);
+            testMetricByName(listMetricStore, PUT_LATENCY_AVG, 1);
```

Comment:
Many of those value changes are due to the fact that we removed `all` tagged parent sensors.